docs/docs/general/metadatastorage.md

```diff
@@ -1,6 +1,6 @@
 # MetadataStorage
 
-The MetadataStorage store all the informations about your decorators, you can get the informations related to them by using `MetadataStorage.instance`
+The MetadataStorage store all the information's about your decorators, you can get the information's related to them by using `MetadataStorage.instance`
```

Comment:
Should be "stores" and "information"?

Reply:
doesn't multiple information called ``informaiton's`` and not ``information``?

Reply:
refactored, review again.